KsApi/models/graphql/GraphComment.swift

```diff
@@ -49,6 +66,18 @@ extension GraphComment {
     self.body = try values.decode(String.self, forKey: .body)
     self.replyCount = try values.nestedContainer(keyedBy: CodingKeys.self, forKey: .replies)
       .decode(Int.self, forKey: .totalCount)
+    self.deleted = try values.decode(Bool.self, forKey: .deleted)
+    self.createdAt = try values.decode(TimeInterval.self, forKey: .createdAt)
+
+    self.authorBadges = []
```

Comment:
If you conform `GraphBadge` to `Decodable` this can just be:

```swift
self.authorBadges = try values.decode(
  [GraphComment.GraphAuthor.GraphAuthorBadges].self,
  forKey: .authorBadges
)
```

Reply:
Yes, this is much simpler. 
I took `GraphBadge` out of author and put it in `GraphComment` as well.

Reply:
Similar change to `Comment`, took `Badge` out of top level. This models the real response more accurately, as mentioned above.